app/src/main/java/com/dhc/dhcandroid/home/HomeViewModel.kt

```diff
@@ -0,0 +1,30 @@
+package com.dhc.dhcandroid.home
+
+import com.dhc.dhcandroid.BaseViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class HomeViewModel @Inject constructor(
+): BaseViewModel<HomeContract.HomeState, HomeContract.HomeEvent, HomeContract.HomeSideEffect>() {
```

Comment:
```suggestion
class HomeViewModel @Inject constructor(
    private val savedStateHandle: SavedStateHandle
): BaseViewModel<HomeContract.HomeState, HomeContract.HomeEvent, HomeContract.HomeSideEffect>() {
```

savedStateHandle 추가하는걸로 하자

Reply:
필요 시에만 사용하는걸루 할게요~!